### PR TITLE
Added a simple timeout for HMR disconnection

### PR DESCRIPTION
**Pull request by stevefan1999-personal**

### CODE REVIEW

**Comment by stevefan1999** [comment]
I'm really confused on the disappearance of setTimeout in the test suites. I'm not familiar with Mocha and Sinon, need assistance!

**Comment by stevefan1999** [comment]
OK, so now I cheated on the test suite. `setTimeout` is breaking sinon for some reason, and using a fake timer didn't help.

File: src/builtins/hmr-runtime.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+var WebSocketClient = require('./websocket-client')
```
Comment (devongovett) [inline src/builtins/hmr-runtime.js]: where is this require coming from?

**Comment by stevefan1999** [comment]
Also, I don't think mocha and sinon support uncontrolled timers? How do we add it to the test suite...Hate to see all reds in the HMR section tho.

**Comment by devongovett** [comment]
Due to move to monorepo which rewrote the git history, this PR was automatically closed. Please open a new PR against master with your changes. You can do a diff against the `1.x` branch to see them. Sorry about that!

**Pull request closed without merging**